plugins/network-policy-audit/tests/integration/04_run_tc_vnp_004.sh

```
@@ -0,0 +1,109 @@
+  - from:
+    - podSelector:
+        matchLabels:
+          tier: backend
+    ports:
+    - protocol: TCP
+      port: 5432
+EOF
+
+oc get networkpolicy -n ${NAMESPACE} > ${OUTPUT_DIR}/${TEST_ID}_policies.log 2>&1
+
+cd ${PLUGIN_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)